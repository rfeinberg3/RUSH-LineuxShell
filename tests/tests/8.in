path /autograder/source/bin
p3.sh & p1.sh & p1.sh
exit
