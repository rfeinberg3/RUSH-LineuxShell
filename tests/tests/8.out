rush> rush> zebra
zebra
cat
rush> 